search/search_quality/search_quality_tool/search_quality_tool.cpp

```diff
@@ -379,6 +379,7 @@ int main(int argc, char * argv[])
   params.m_locale = FLAGS_locale;
   params.m_numThreads = FLAGS_num_threads;
   TestSearchEngine engine(move(infoGetter), make_unique<ProcessorFactory>(), Engine::Params{});
+  engine.SetLocale(FLAGS_locale);
```

Comment:
Why do we need this?

Reply:
This looks like the only way to set a locality lang now.

Reply:
But search query has locality field too. What this change fixes?

Reply:
The correct change was to pass params to the engine constructor.
fixed